src/models/book.js

```diff
@@ -0,0 +1,27 @@
+module.exports = (connection, DataTypes) => {
+  const schema = {
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: { args: true, msg: "Please enter a title" },
+        notNull: { args: true, msg: "Please enter a title" },
+      },
+    },
+    author: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: { args: true, msg: "Please enter an author" },
+        notNull: { args: true, msg: "Please enter an author" },
+      },
+    },
```

Comment:
This should be removed as you have used the Sequelize associations to create the foreign key authorId on the book table, so you wouldn't want to also manually type in the author again. If you check you schemas in PGAdmin you will see what columns and constraints you have on each table